openshift/release/knative-eventing-channelbroker-ci.yaml

```diff
@@ -160,3 +160,165 @@ spec:
+    resources:
+      - "namespaces/finalizers"
+    verbs:
+      - "update"
+  - apiGroups:
+    - coordination.k8s.io
+    resources:
+    - leases
+    verbs:
+    - "get"
+    - "list"
+    - "create"
+    - "update"
+    - "delete"
+    - "patch"
+    - "watch"
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: configmappropagations.configs.internal.knative.dev
+  labels:
+    eventing.knative.dev/release: devel
+    knative.dev/crd-install: "true"
+spec:
+  group: configs.internal.knative.dev
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  names:
+    kind: ConfigMapPropagation
+    plural: configmappropagations
+    singular: configmappropagation
+    categories:
+      - knative-internal
+    shortNames:
+      - kcmp
+      - cmp
+  scope: Namespaced
+  subresources:
+    status: {}
+  additionalPrinterColumns:
+    - name: Ready
+      type: string
+      JSONPath: ".status.conditions[?(@.type==\"Ready\")].status"
+    - name: Reason
+      type: string
+      JSONPath: ".status.conditions[?(@.type==\"Ready\")].reason"
+    - name: OriginalNamespace
+      type: string
+      JSONPath: ".spec.originalNamespace"
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+            - originalNamespace
+          properties:
+            originalNamespace:
+              type: string
+              description: "The namespace where original ConfigMaps exist in."
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: broker-controller
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: broker-controller
+  template:
+    metadata:
+      labels:
+        app: broker-controller
+        eventing.knative.dev/release: devel
+    spec:
+      serviceAccountName: eventing-controller
+      containers:
+      - name: broker-controller
+        terminationMessagePolicy: FallbackToLogsOnError
+        image: registry.svc.ci.openshift.org/openshift/knative-v0.15.0:knative-eventing-channel-broker
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
+        env:
+          - name: SYSTEM_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: CONFIG_LOGGING_NAME
+            value: config-logging
+          - name: CONFIG_OBSERVABILITY_NAME
+            value: config-observability
+          - name: METRICS_DOMAIN
+            value: knative.dev/eventing
+          - name: BROKER_INGRESS_IMAGE
+            value: registry.svc.ci.openshift.org/openshift/knative-v0.15.0:knative-eventing-ingress
+          - name: BROKER_INGRESS_SERVICE_ACCOUNT
+            value: eventing-broker-ingress
+          - name: BROKER_FILTER_IMAGE
+            value: registry.svc.ci.openshift.org/openshift/knative-v0.15.0:knative-eventing-filter
```

Comment:
one more manual thing.

looking here: https://github.com/openshift/release/pull/9765/files#diff-627940ffc7041a4e56cb09b23f41c26aR35

these two will be build/pused as:
* `knative-eventing-broker-filter`
* `knative-eventing-broker-ingress`